SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/TestimoniosCollectionViewCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TestimoniosCollectionViewCell.swift
+//  SomosMasApp
+//
+//  Created by Fran Nasich on 01/04/2022.
+//
+
+import UIKit
+
+class TestimoniosCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var testimonioImage: UIImageView!
```

Comment:
naming is an important thing, all the app is in English why you name your clases in Spanish???? remember that this is a public repository any one can see it

Reply:
Ok